core/store/migrations/0022_add_confirmations_to_direct_request.go

```diff
@@ -0,0 +1,27 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+const (
+	up22 = `
+ALTER TABLE direct_request_specs ADD COLUMN num_confirmations bigint DEFAULT 1 NOT NULL;
```

Comment:
How about leaving default NULL and falling back to the env var `MIN_INCOMING_CONFIRMATIONS` if not set?

Reply:
yes, changed that to NULL now, so the following should still work fine:

```
minConfirmations := d.config.MinRequiredOutgoingConfirmations()
	if concreteSpec.NumConfirmations > minConfirmations {
		minConfirmations = concreteSpec.NumConfirmations
	}
```
as in case of NULL, concreteSpec.NumConfirmations will be 0 as far as i understand how gorm works

Reply:
Hmm. If it's nullable we should use `null.Int64` or something like that.

Reply:
ok used null.Uint32